src/app/public/modules/modal/modal.component.scss

```diff
@@ -58,7 +58,6 @@
 
 .sky-modal-content {
   background-color: #fff;
-  z-index: 1;
```

Comment:
How can we be sure this doesn't introduce other visual bugs?

Reply:
I chatted with Conor about this on Friday and he doesn’t really remember why this was added when it was. He originally added this in a PR that dealt with auto focusing on elements in the modal. I was able to verify that removing this does not introduce a regression there. Other than that it shouldn’t introduce any other visual bugs as this was pulling the content section into its own stacking context relative to the footer and header but anything inside of the content area would still be using stacking contexts relative to each other. 

This was causing the footer and content to not be relative to each other which meant that a content piece with z-index of 99999 could still show up behind a footer element with z-index of 2. 

I totally see where we can’t predict everything here but see as this was added in a non-visual fix originally and it will only affect things that overflow from the content I think we are pretty safe here in general.

Reply:
Ah, I remember what it was for now... It was to show focus on the modal content. It should have a blue outline when the modal pops up, and doesn't have any focusable content inside it. Does this change the way that works?

Reply:
I'll check

Reply:
Chatted with Steve about this and made a small change to ensure everything looks good after this change. I think we are good for another review now :)